### DISCUSSION TITLE
Connect iOS device to Windows for debugging?

### DISCUSSION THREAD

**Discussion by perholmes** [airsdk/Adobe-Runtime-Support#2290] [General]
I'd like to be able to debug on an iOS device directly from the Windows IDE. It still seems like it's supposed to be supported, and the iPad does show up in Device Manager and Windows Explorer.

But adt/idb commands still don't see the device:

```
$ ./adt -devices -platform ios
No connected device found.

$ ./idb.exe -devices
No connected device found.
```

Does anyone know what I'm missing?

**Comment by ajwfrost** [comment]
Possibly iTunes? I had been finding recently that I needed to launch iTunes which I think triggers some extra libraries or connectivity or something, upon which the idb process depends.

But actually, before you do that, can you check whether it works if you run the "ideviceinfo.exe" application from that folder? `idb` does some cunning usb thing to start with to see how many devices are available and then calls these other utility apps for more information, but I had been wondering if we can do away with the usb detection part and just rely on `ideviceinfo` etc. So if you find that `ideviceinfo` works but `idb` doesn't, it would indicate we need to do a bit of rework....

thanks

**Comment by perholmes** [comment]
OK, I wasn't sure if it was still with iTunes. Last version is years old. I'll work on Mac for a bit, need to work on Storyboards anyway. Thanks!

**Comment by marchbold** [comment]
If you haven't authorised the usb connection and enabled developer mode on some of the most recent iOS devices then you won't see any devices in the list. Try making sure you have been through that before attempting to connect from AIR.
